_sections/60-Ressources.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Resources
+bg: lightgray
+color: black
+style: center
+fa-icon: fa-wrench
+---
+
+## JabRef-compatible text editors
+
+JabRef can push entries, i.e. insert `\cite{key}` commands, to the following text editors:
+
+- [Emacs](https://www.gnu.org/software/emacs/)
+- [LaTeX Editor](http://www.latexeditor.org/)
+- [LyX/Kile](http://www.lyx.org/)
+- [TeXstudio](http://www.texstudio.org/)
+- [Texmaker](http://www.xm1math.net/texmaker/)
+- [Vim](http://www.vim.org/)
+- [WinEdt](http://www.winedt.com/)
+
+Additionally, JabRef can natively insert citations and format a bibliography in:
+
+- [OpenOffice Writer](https://www.openoffice.org/)
+- [LibreOffice Writer](https://www.libreoffice.org/)
+
+See [OpenOffice/LibreOffice integration](https://help.jabref.org/en/OpenOfficeIntegration) for details.
+
+## External tools
+
+### JabFox
+
+_by Tobias Diez_
+
+JabFox is a Firefox add-on for users of the bibliographic reference manager JabRef. It automatically identifies and extracts bibliographic information on websites and sends them to JabRef in one click. A wide range of publisher sites, library catalogs and databases are supported.
+
+[Firefox Addon](https://addons.mozilla.org/en-US/firefox/addon/jabfox/)
+
+### WinEdt's JabRef launcher
+
+_by Karl Koeller_
+
+This WinEdt's package allows to launch the JabRef program from within WinEdt.
+
+[WinEdt's JabRef launcher](http://www.winedt.org/config/menus/JabRef.html)
+
+### Export-Filter Editor
+
+_by Felix Wilke_
+
+Using this tool you can easily create a custom export filter for JabRef to build you own bibliography style.
+
+It supports:
+
+*   HTML Export Filter
+*   RTF Export Filter
+*   OpenOffice/ LibreOffice Style File
+*   Saving the filter for later refinements
+
+[Export-Filter Editor](https://sourceforge.net/projects/efe/?source=dlp)
+
+### PocketBib
+
+_by Graham Dennis_
+
+PocketBib is a BibTeX library reader for academics on the go. With PocketBib you can search, browse and read the papers in your BibTeX library, and sync them with BibDesk or JabRef on your computer.
+
+Available for iPad, iPhone and iPod Touch.
+
+[PocketBib](http://www.grahamdennis.me/pocketbib/)
+
+### Bibtex4word
+
+_by Mike Brookes_
+
+Bibtex4Word is an add-in for Microsoft Word that allows the citation of references and the insertion of a bibliography into your document using your choice of formatting style. It is lightweight, transparent and does not mess up your documents.
+
+[Bibtex4Word](http://www.ee.ic.ac.uk/hp/staff/dmb/perl/index.html)
+
+### Feinerleiser
+
+**!!!Status?**
+
+_by Niklas Alt_
+
+Feinerleiser is a tool for improving the JabRef-LibreOffice integration when writing for the Humanities. This tool can be run to finalize a document, providing citation features that are not supported by JabRef itself.
+
+[Feinerleiser](http://www.sourceforge.net/projects/feinerleiser/)
```

Comment:
As the last update is from 2013, I doubt that it is still actual. But when I see it right it as external tool, which does not depend on Jabref itself but only on OO and